client/src/modules/list/model/actionTypes.js

```diff
@@ -35,6 +35,7 @@ export const ListActionTypes = Object.freeze({
   LEAVE_FAILURE: 'list/LEAVE_FAILURE',
   LEAVE_SUCCESS: 'list/LEAVE_SUCCESS',
   REMOVE_ARCHIVED_META_DATA: 'list/REMOVE_ARCHIVED_META_DATA',
+  REMOVE_BY_IDS: 'list/REMOVE_BY_IDS',
```

Comment:
this is probably the same issue as on the other branch that I commented yesterday
If this action means that user is gonna be removed from many lists this should be named like REMOVE_FROM_LISTS or even REMOVE_FROM_MANY 

Decide which one or give your proposition

Reply:
This event removes lists. If someone removes you from the cohort and you are on the dashboard you have to lose this cohort's list. This event sends an array of lists' ids which have to be removed from the dashboard, we don't use 'LIST' word in List action types names so I think this action's type can be named as ListActionTypes.REMOVE_BY_IDS